readme.md

```diff
@@ -3,18 +3,18 @@
 _enforce a delay on code execution to give affected parties time to react_
 
 `ds-pause` allows authorized entities to make `plans`. A `plan` describes a single `delegatecall`
-operation and a unix timestamp `era` before which it cannot be executed.
+operation and a unix timestamp `eta` before which it cannot be executed.
 
-Once `era` has passed, a `plan` can be executed by anyone.
+Once `eta` has passed, a `plan` can be executed by anyone.
 
-A `plan` can only be made if it's `era` is after `block.timestamp + delay`. The `delay` is
+A `plan` can only be made if it's `eta` is after `block.timestamp + delay`. The `delay` is
 configurable upon construction.
 
 A `plan` consists of:
 
 - `usr`: the address to `delegatecall` into
 - `fax`: the `calldata` to use
-- `era`: the time before which the `plan` cannot be executed
+- `eta`: the time before which the `plan` cannot be executed
```

Comment:
Might be worth turning this into a positive sentence: "the time after which the `plan` can be executed"

Reply:
done: "The time from when the `plan` can be executed"